src/main/java/stringcalculator/StringCalculator.java

```diff
@@ -0,0 +1,49 @@
+package stringcalculator;
+
+import org.springframework.util.StringUtils;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private static final int MIN_TEXT_LENGTH = 1;
+    private static final String DEFAULT_DELIMITER = "[,:]";
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+
+    public int add(final String text) {
+        if (!StringUtils.hasText(text)) {
+            return 0;
+        }
+        if (isTextMinLength(text)) {
+            return Integer.parseInt(text);
+        }
+        return sumTokens(getTokens(text));
+    }
+
+    private String[] getTokens(final String text) {
+        Matcher m = Pattern.compile(CUSTOM_DELIMITER).matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            return m.group(2).split(customDelimiter);
+        }
+        return text.split(DEFAULT_DELIMITER);
+    }
+
+    private boolean isTextMinLength(String text) {
+        return text.length() == MIN_TEXT_LENGTH;
+    }
+
+    private int sumTokens(final String[] tokens) {
+        return Arrays.stream(tokens).mapToInt(this::parseNonNegative).sum();
```

Comment:
```suggestion
        return Arrays.stream(tokens)
        .mapToInt(this::parseNonNegative)
        .sum();
```
스트림은 가독성을 위해 기능단위로 줄바꿈하는것을 권장 드려요 😄

Reply:
IntelliJ에서 cmd+option+L을 눌렀더니 저렇게 되어버렸네요..!! 설정 바꾸는 방법을 찾아보도록 하겠습니다 ㅎㅎ